app/controllers/api/subcollections/schedules.rb

```diff
@@ -2,7 +2,19 @@ module Api
   module Subcollections
     module Schedules
       def schedules_query_resource(object)
-        object ? object.list_schedules : {}
+        object.miq_schedules
```

Comment:
won't miq_schedules behave differently than list_schedules ?    report vs service template schedules ?

Reply:
yes, I moved the reporting specific method to the reports controller and removed the `include` for the schedules subcollection